content/id/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -0,0 +1,186 @@
+reviewers:
+- freehan
+title: EndpointSlice
+feature:
+  title: EndpointSlice
+  description: >
+    Pelacakan _endpoint_ jaringan yang _scalable_ pada klaster Kubernetes.
+
+content_template: templates/concept
+weight: 15
+---
+
+
+{{% capture overview %}}
+
+{{< feature-state for_k8s_version="v1.17" state="beta" >}}
+
+_EndpointSlice_ menyediakan sebuah cara yang mudah untuk melacak _endpoint_ jaringan dalam sebuah
+klaster Kubernetes. _EndpointSlice_ memberikan alternatif yang lebih _scalable_ dan lebih dapat diperluas daripada Endpoint.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Motivasi
+
+Endpoint API telah menyediakan sebuah cara yang mudah dan sederhana untuk
+melacak _endpoint_ jaringan pada Kubernetes. Sayangnya, seiring dengan besarnya klaster Kubernetes
+dan Service, batasan-batasan yang dimiliki API tersebut semakin terlihat.
+Terutama, hal tersebut termasuk kendala-kendala mengenai proses _scaling_ _endpoint_ jaringan
+dalam jumlah yang besar.
+
+Karena semua _endpoint_ jaringan untuk sebuah Service disimpan dalam satu sumber daya 
+Endpoint, sumber daya tersebut dapat menjadi cukup besar. Hal itu dapat mempengaruhi performa
+dari komponen-komponen Kubernetes (terutama _master control plane_) dan menyebabkan
+trafik jaringan dan pemrosesan yang cukup besar ketika Endpoint berubah.
+EndpointSlice membantu kamu menghindari masalah-masalah tersebut dan juga menyediakan _platform_
+yang dapat diperluas untuk fitur-fitur tambahan seperti _topological routing_.
+
+## Sumber daya EndpointSlice {#endpointslice-resource}
+
+Pada Kubernetes, sebuah EndpointSlice memiliki referensi-referensi terhadap serangkaian _endpoint_
+jaringan. _Controller_ EndpointSlice secara otomatis membuat EndpointSlice
+untuk sebuah Kubernetes Service ketika sebuah {{< glossary_tooltip text="_selector_"
+term_id="selector" >}} dituliskan. EndpointSlice tersebut akan memiliki
+referensi-referensi menuju Pod manapun yang cocok dengan _selector_ pada Service. EndpointSlice mengelompokkan
+_endpoint_ jaringan berdasarkan kombinasi Service dan Port yang unik.
+Nama dari sebuah objek EndpointSlice haruslah berupa
+[_DNS subdomain name_](/docs/concepts/overview/working-with-objects/names#dns-subdomain-names) yang sah.
+
+Sebagai contoh, berikut merupakan sampel sumber daya EndpointSlice untuk sebuah Kubernetes Service
+yang bernama `example`.
+
+```yaml
+apiVersion: discovery.k8s.io/v1beta1
+kind: EndpointSlice
+metadata:
+  name: example-abc
+  labels:
+    kubernetes.io/service-name: example
+addressType: IPv4
+ports:
+  - name: http
+    protocol: TCP
+    port: 80
+endpoints:
+  - addresses:
+      - "10.1.2.3"
+    conditions:
+      ready: true
+    hostname: pod-1
+    topology:
+      kubernetes.io/hostname: node-1
+      topology.kubernetes.io/zone: us-west2-a
+```
+
+Secara _default_, setiap EndpointSlice yang dikelola oleh _controller_ EndpointSlice tidak akan memiliki
+lebih dari 100 _endpoint_. Di bawah skala tersebut, EndpointSlice akan memetakan 1:1
```

Comment:
```suggestion
lebih dari 100 _endpoint_. Dibawah skala tersebut, EndpointSlice akan memetakan 1:1
```

Reply:
Shouldn't the former be right to show preposition?

Reply:
Ah i was wrong, sorry

Reply:
Alright!